### DISCUSSION TITLE
Filters question

### DISCUSSION THREAD

**Discussion by DenverDaveH** [Xyrio/RSSOwlnix#148] [Q&A]
Hello,
Firstly, thanks for the killer app!!
Secondly, is there a way of filtering Reddit posts? I remember reading somewhere long ago, that if a user made a post in 1 sub, that you can't filter it in a 2nd sub due to something in the Reddit code that made each post unique.

Is there a way to filter out by name of the poster? So if a user posts in Sub1 and Sub2, assuming it is the same post based on the nature of the sub, if a post is found in Sub2 etc, how can we avoid seeing the same post in subsequent subs?

Thanks again!

**Comment by Xyrio** [comment]
not sure what you mean.

rssowlnix can mark news as read that point to same article.  menu - tools - preferences - feeds - reading - automatically mark duplicate news read

if you need something custom you could do it manually using a search on a folder with just the feeds you want to compare then do a search on that folder for everything unread, sort afterwards by user and remove what you consider duplicates.

the trick is basically to do a search so you have what you want in one list. then sort in some way so that duplicates come in order and you can see them easily.

**Reply by DenverDaveH** [reply]
I have that checked. Yet for some Reddit subs, if someone posts something, I see that post in other subs. Sometimes is a different subject line but same content/pic. Sometimes its the same subject line with different pic. All by the same author.

Any thoughts?

**Reply by Xyrio** [reply]
if the link to the article is not the same it dosnt work. then you need to do it manually.

if the titles are the same you can sort by that. or if nothing else the modified date should be the same if it is really the same article?
